sqlite3/lib/src/wasm/statement.dart

```diff
@@ -161,6 +161,8 @@ class WasmStatement extends CommonPreparedStatement {
   void _bindParam(Object? param, int i) {
     if (param == null) {
       bindings.sqlite3_bind_null(statement, i);
+    } else if (param is int) {
+      bindings.sqlite3_bind_int64(statement, i, BigInt.from(param));
```

Comment:
Same here, it would be good to test this and ensure doubles still work properly.